deploy/pom.xml

```diff
@@ -521,6 +521,16 @@
             <artifactId>httpmime</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+        <groupId>org.apache.httpcomponents.core5</groupId>
+        <artifactId>httpcore5</artifactId>
+        <scope>compile</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents.client5</groupId>
+        <artifactId>httpclient5</artifactId>
+        <scope>compile</scope>
+      </dependency>
```

Comment:
Whats the reason for moving these to shared?

Reply:
httpclient has been in shared since @jonespm 

SAK-33388 - Update httpclient/httpcore to newest, have all tools use shared version (#4813)